app/controllers/products_controller.rb

```diff
@@ -9,36 +9,34 @@ def new
     @product = Product.new
   end
 
-  def show
-  end
+  def show; end
 
-  def edit
-  end
+  def edit; end
 
   def create
     @product = Product.new(product_params)
-    respond_to do|format|
```

Comment:
remove extra code of respond to
write like
`if @product.save
    redirect_to @product, notice: 'product was successfully created'
else
   render :new
end